pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -608,6 +599,17 @@ func (npm *nominatedPodMap) AddNominatedPod(pod *v1.Pod, nodeName string) {
 	npm.Unlock()
 }
 
+// NominatedPodsForNode returns pods that are nominated to run on the given node,
+// but they are waiting for other pods to be removed from the node before they
+// can be actually scheduled.
```

Comment:
I know it's a copy and paste, but `actually` does not add anything to the sentence.

Reply:
For English wordings, I must trust you :)